CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Change Log
 
+## Version 3.2.0 *(2020-04-02)*
+
+This is a new minor release with numerous internal changes.
+
+### Summary of changes
+
+* Chucker won't load the whole response into memory anymore, but will mutlicast it with the help of temporary files. It allows to avoid issues with OOM, like in reported in [#218].
+This change also allows to avoid problems with Chucker consuming responses, like reported in [#242].
+* Added a red open padlock icon to clearly indicate HTTP requests in transactions list.
+* Added TLS info (version and cipher suite) into `Overview` tab.
+* Added ability to encode/decode URLs.
+* Added RTL support.
+* Switched from AsyncTasks to Kotlin coroutines.
+* Switched to [ViewBinding](https://developer.android.com/topic/libraries/view-binding).
+* Bumped targetSDK to 29.
+* Greatly increased test coverage (we will add exact numbers and reports pretty soon).
+
+### Bugfixes
+
+* Fix for [#218] with OOM exceptions on big responses.
```

Comment:
The links need to be added at the bottom of the Markdown file otherwise they won't work (there are already some)

Reply:
Done